apps/frontend/src/App.tsx

```diff
@@ -33,6 +33,23 @@ init({
   // for finer control
   tracesSampleRate: 1.0,
   attachStacktrace: true,
+  // Turn sampleRate on to reduce the amount of data sent to Sentry
+  // sampleRate: 0.1, // 0.1 = 10% of error events will be sent
+  allowUrls: [
+    'hub.asap.science/static/js/', // your code
+    'asap-hub.us.auth0.com', // code served from Auth0
+    'hub.asap.science/.auth/static/js', // code served from Auth0
+  ],
+  denyUrls: [
+    /fonts\.googleapis\.com/i, // code served from Google Fonts
+    /extensions\//i, // Chrome extensions
+    /^chrome:\/\//i, // Chrome extensions
+    /gtag\/js\//i, // Google Tag Manager
+  ],
+  ignoreErrors: [
+    /Failed to fetch \w+.*? list. Expected status 2xx. Received status 401/g,
+    /Loading chunk [0-9]*? failed/g,
```

Comment:
Some of the chunks are named eg main. They won't be picked up by this regex. Check the built files for all the examples

Reply:
Let's go with this, and see if we have any issues with "named" chunks. I guess not, because the error I saw on Sentry was for a named chunk, and it appeared as a number (welcome.chunk gets us: Loading chunk 55 failed.)